src/modules/Exchange.ts

```diff
@@ -27,10 +30,14 @@ export class Exchange {
     this.contracts = contracts;
   }
 
-  public getAddress(): string {
+  public getStarkwareAddress(): string {
```

Comment:
I think we should revert this line, I think it's a breaking and unnecessary change